contracts/LoanManager.sol

```diff
@@ -19,34 +19,34 @@ import "./MonetarySupervisor.sol";
 contract LoanManager is Restricted, TokenReceiver {
     using SafeMath for uint256;
 
-    enum LoanState { Open, Repaid, Defaulted, Collected } // NB: Defaulted state is not stored, only getters calculate
+    enum LoanState { Open, Repaid, DoNotUse, Collected } // NB: DoNotUse state is kept for backwards compatibility only (so the ordinal of 'Collected' does not shift), as the name states: do not use it.
 
     struct LoanProduct {
-        uint minDisbursedAmount; // 0: with decimals set in AugmintToken.decimals
-        uint32 term;            // 1
-        uint32 discountRate;    // 2: discountRate in parts per million , ie. 10,000 = 1%
-        uint32 collateralRatio; // 3: loan token amount / colleteral pegged ccy value
-                                //      in parts per million , ie. 10,000 = 1%
-        uint32 defaultingFeePt; // 4: % of collateral in parts per million , ie. 50,000 = 5%
-        bool isActive;          // 5
+        uint minDisbursedAmount;    // 0: minimum loanAmount, with decimals set in AugmintToken.decimals (i.e. token amount)
+        uint32 term;                // 1: term length (in seconds)
+        uint32 discountRate;        // 2: discountRate (in parts per million, i.e. 10,000 = 1%)
+        uint32 collateralRatio;     // 3: inverse of collateral ratio: [repayment value (in token) / collateral value (in token)] (in ppm). Note: this is actually the inverse of the commonly used "collateral ratio"! TODO: fix it
```

Comment:
Would it a be a big deal to fix it now? Or Separate PR?

Reply:
I suggest addressing this separately, the goal of this PR is to add the basic margin call in the least invasive way.

Reply:
I'd really like to fix this before the deploy, but i'm ok with doing it in a separate PR.